docs/google.md

```diff
@@ -127,6 +129,24 @@ process predefined_resources_task {
 }
 ```
 
+:::{versionadded} 23.06.0-edge
+:::
+
+The `disk` directive can be used to set the boot disk size or provision a disk for scratch storage. If the disk type is specified with the `type` option, a new disk will be mounted to the task VM at `/tmp` with the requested size and type. Otherwise, it will set the boot disk size, overriding the `google.batch.bootDiskSize` config option. See the [Google Batch documentation](https://cloud.google.com/compute/docs/disks) for more information about the available disk types.
+
+Examples:
+
+```groovy
+// set the boot disk size
+disk 100.GB
+
+// mount a persistent disk at '/tmp'
```

Comment:
```suggestion
// mount a local disk at '/tmp'
```

Reply:
It should say "persistent" here because the disk type is `pd-standard`. The following example uses a local disk